minimal-notebook/jupyter_notebook_config.py

```diff
@@ -8,8 +8,8 @@
 PEM_FILE = os.path.join(jupyter_data_dir(), 'notebook.pem')
 
 c = get_config()
-c.NotebookApp.ip = os.getenv('INTERFACE', '') or '*'
-c.NotebookApp.port = int(os.getenv('PORT', '') or 8888)
+c.NotebookApp.ip = '*'
+c.NotebookApp.port = 8888
```

Comment:
Cool. Makes sense to just standardize on these here so we can reason about them in the orchestration systems.